extensions/default/src/commandsModule.ts

```diff
@@ -395,6 +395,7 @@ const commandsModule = ({
           hangingProtocolService.setProtocol(protocolId, {
             stageId,
             stageIndex: useStageIdx,
+            displaySetSelectorMap,
```

Comment:
I think this is the correct thing here because since we are reapplying the hp (and stage) we should also apply the same display set that it might be displaying.